### PR TITLE
image build target additions

### PR DESCRIPTION
**Pull request by adrianchiris**
- Add webhook to image build target
- use APP_NAME var in image name for consistency

### CODE REVIEW

**Comment by adrianchiris** [comment]
@SchSeba @zeeke PTAL

File: Makefile [hunk 1]
```diff
@@ -14,14 +14,16 @@ IMAGE_BUILDER?=docker
...
+DOCKERFILE_WEBHOOK?=Dockerfile.webhook
 
 CRD_BASES=./config/crd/bases
 
 export APP_NAME?=sriov-network-operator
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
 IMAGE_REPO?=ghcr.io/k8snetworkplumbingwg
 IMAGE_TAG?=$(IMAGE_REPO)/$(APP_NAME):latest
-CONFIG_DAEMON_IMAGE_TAG?=$(IMAGE_REPO)/sriov-network-config-daemon:latest
+CONFIG_DAEMON_IMAGE_TAG?=$(IMAGE_REPO)/$(APP_NAME)-config-daemon:latest
```
Comment (vasrem) [inline Makefile:25]: Notice that this is changing from:
```
sriov-network-config-daemon
```
to:
```
sriov-network-operator-config-daemon
```

Maybe other things need to be updated as well. https://github.com/search?q=repo%3Ak8snetworkplumbingwg%2Fsriov-network-operator%20sriov-network-config-daemon&type=code

Comment (adrianchiris) [reply Makefile:25]: image which is built part of our pipeline is named sriov-network-operator-config-daemon

see: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pkgs/container/sriov-network-operator-config-daemon

the goal is to align makefile target to tag with same image name as we publish.

not aiming into renaming the component ATM :)

**Review by SchSeba — APPROVED** [review]
LGTM

**Pull request merged**